db-checkbook/build_scripts/01_dataloading.py

```diff
@@ -1,16 +1,39 @@
-from dcpy.connectors.s3 import client
+#from dcpy.connectors.s3 import client
```

Comment:
going off of another comment I think that eventually we'll incorporate the client function from dcpy to grab the data from digital ocean